src/main/webapp/app/app.component.ts

```diff
@@ -1,20 +1,24 @@
-import { ChangeDetectionStrategy, Component, Inject, OnInit } from '@angular/core';
-import { Observable } from 'rxjs';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Inject,
+  OnInit,
+} from "@angular/core";
+import { Observable } from "rxjs";
 
-import { Task } from './tasks/task';
-import { TaskService } from './tasks/task.service';
+import { Task } from "./tasks/task";
+import { TaskService } from "./tasks/task.service";
 
 @Component({
-  selector: 'tiny-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush
+  selector: "tiny-root",
```

Comment:
unfortunately this breaks our linting

Reply:
These syntax changes are because of my prettier config. Maybe it is good idea to save the prettier config in the app.

Reply:
You mean the double quotes? Right. I do not know how this happened. I should have reviewed the commit before pushing.